config/crds/migration_v1alpha1_directvolumemigration.yaml

```diff
@@ -61,6 +61,11 @@ spec:
               type: array
             failedPods:
               items:
+                properties:
+                  lastObservedProgressPercent:
```

Comment:
NIT: `lastObservedProgressPercentage`?

Don't really have a strong opinion either way just my thought.

Reply:
@dymurray how about just `progress`?

Reply:
and `transferRate`? the `lastObserved` is obvious now that I am actually thinking while naming :)

Reply:
yeah I like just `progress` and `transferRate`